.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
I don't like putting .envrc in the repo. It means that anyone using direnv and wanting a different enviroment/not use the flake env has a hard time.

Reply:
As an alternative, I suggest adding .envrc to gitignore so people can easily add a envrc themselves

Reply:
All right.